livingatlas/pipelines/src/main/java/au/org/ala/clustering/OccurrenceRelationships.java

```diff
@@ -0,0 +1,251 @@
+package au.org.ala.clustering;
```

Comment:
I foresee this will quickly become a mess if we fork code like this. These are slightly modified versions of the GBIF codebase and will deviate.

Wouldn't it be better to have common code to work on?

Reply:
Fully agree. Not sure how best to proceed without refactoring the original `occurrence-clustering` module, removing the use of Spark artefacts.

First thought is that `OccurrenceRelationships` logic should be dealing with POJOs without Spark, AVRO etc. and perhaps this should be moved out to a separate library.